prov/usnic/src/fi_ext_usnic.h

```diff
@@ -44,11 +44,19 @@
 
 #define FI_PROTO_RUDP (100U | FI_PROV_SPECIFIC)
 
-#define FI_EXT_USNIC_INFO_VERSION 1
+#define FI_EXT_USNIC_INFO_VERSION 2
+
+#define USDF_MAX_DEVNAME 16
+#define USDF_CAP_MAX 5
```

Comment:
I'd prefer all public names in this extension header to start with `FI_EXT_USNIC_` / `FI_USNIC_` / `fi_usnic_`.  Let's keep `USDF_` / `usdf_` names internal to the usnic provider.